rest-api/recipe/models/recipe-models.js

```diff
@@ -140,7 +189,7 @@ async function getRecipeById(id) {
   return { ...recipe, tags, categories, images, instructions, ingredients };
 }
 
-async function addRecipeTransaction(body) {
+async function addRecipeTransaction(body, parent = false) {
```

Comment:
I'm not sure I understand what `parent` is for, here (and below). Could you explain briefly? :)

Reply:
Yes good question, the default use of addRecipeTransaction is to create a new Recipe, following that logic a new recipe never has a parent, hence false and by default set the parent_id to itself.
When parent gets passed as true (in the case of cloning), it then passes the conditional that it will not change the parent_id which already points to a previous recipe

Reply:
Ah, gotcha!